src/agent/src/message_queue_utils.hpp

```diff
@@ -15,3 +17,9 @@ boost::asio::awaitable<std::string> GetMessagesFromQueue(std::shared_ptr<IMultiT
 void PopMessagesFromQueue(std::shared_ptr<IMultiTypeQueue> multiTypeQueue, MessageType messageType);
 
 void PushCommandsToQueue(std::shared_ptr<IMultiTypeQueue> multiTypeQueue, const std::string& commands);
+
+std::optional<command_store::Command> GetCommandFromQueue(std::shared_ptr<IMultiTypeQueue> multiTypeQueue);
+
+bool PopCommandFromQueue(std::shared_ptr<IMultiTypeQueue> multiTypeQueue);
+
+int DispatchCommand(command_store::Command&);
```

Comment:
This is not a message queue util. Maybe we can create `command_utils.hpp` and `command_utils.cpp` to place this.

Reply:
Done